environments/prod/hmcts-net.yml

```diff
@@ -253,6 +253,10 @@ txt:
     ttl: 14400
     record:
       - "v=TLSRPTv1;rua=mailto:tls-rua@mailcheck.service.ncsc.gov.uk"
+  - name: _dmarc
```

Comment:
@NickAzureDevops this has already defined above

Reply:
yeh i saw that